specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -4168,486 +4176,590 @@
-			},
-			"httpHeaders": {
-				"$ref": "#/definitions/WebApiHttpHeaders",
-				"description": "The headers required to make the http request."
-			},
-			"httpMethod": {
-				"type": "string",
-				"description": "The method for the http request."
-			},
-			"timeout": {
-				"type": "string",
-				"format": "duration",
-				"description": "The desired timeout for the request. Default is 30 seconds."
-			},
-			"batchSize": {
-				"type": "integer",
-				"format": "int32",
-				"x-nullable": true,
-				"description": "The desired batch size which indicates number of documents."
-			}
-		},
-		"required": [
-          "uri", "httpHeaders", "httpMethod"
-        ],
-		"externalDocs": {
-			"url": "https://docs.microsoft.com/azure/search/cognitive-search-custom-skill-interface"
-	    },
-		"description": "A skill that can call a Web API endpoint, allowing you to extend a skillset by having it call your custom code."
-	},
-	"WebApiHttpHeaders": {
-		"properties": {
-			"headers": {
-				"type": "object",
-				"additionalProperties": {
-					"type": "string"
-				},
-				"description": "A dictionary of http request headers."
-			}
-		}
-	},
-	"SkillsetListResult": {
-		"properties": {
-			"value": {
-				"x-ms-client-name": "Skillsets",
-				"type": "array",
-				"readOnly": true,
-				"items": {
-					"$ref": "#/definitions/Skillset"
-				},
-				"description": "The skillsets defined in the Search service."
-			}
-		},
-		"description": "Response from a list Skillset request. If successful, it includes the full definitions of all skillsets."
-	},
-	"TextExtractionAlgorithm": {
-		"type": "string",
-        "enum": [
-          "printed",
-          "handwritten"
-        ],
-        "x-ms-enum": {
-          "name": "TextExtractionAlgorithm",
-          "modelAsString": false
-        },
-		"description": "A value indicating which algorithm to use. Default is printed."
-	},
-	"TextSplitMode": {
-		"type": "string",
-        "enum": [
-          "pages",
-          "sentences"
-        ],
-        "x-ms-enum": {
-          "name": "TextSplitMode",
-          "modelAsString": false
-        },
-		"description": "A value indicating which split mode to perform."
-	},
-	"VisualFeature": {
-		"type": "string",
-        "enum": [
-          "categories",
-          "tags",
-		  "description",
-		  "faces",
-		  "imageType",
-		  "color"
-        ],
-        "x-ms-enum": {
-          "name": "VisualFeature",
-          "modelAsString": false
-        },
-		"description": "The strings indicating what visual feature types to return."
-	},
-	"ImageDetail": {
-		"type": "string",
-        "enum": [
-          "celebrities",
-          "landmarks"
-        ],
-        "x-ms-enum": {
-          "name": "ImageDetail",
-          "modelAsString": false
-        },
-		"description": "A string indicating which domain-specific details to return."
-	},
-	"NamedEntityCategory": {
-		"type": "string",
-        "enum": [
-          "location",
-          "organization",
```

Comment:
It's very difficult to tell what changed here. Please separate formatting changes into a separate commit.